### ISSUE TITLE
Daily ML Card Review

### ISSUE THREAD

**Issue by CLuiz** [Starsa/Mentorship#7]
Review at least one card every morning. List the card(s) reviewed daily in issue comments, take notes and ask questions in this card for future discussion.

### Acceptance Criteria
- [ ] Monday
- [x] Tuesday
- [x] Wednesday
- [ ] Thursday
- [ ] Friday
- [ ] Saturday
- [ ] Sunday

**Comment by Starsa** [comment]
**Accuracy-** I have been asked what other metrics can evaluate a model, and how do you explain it

**Comment by Starsa** [comment]
**Bias Variance Tradeoff**- I understand, but on a high level. If we could drill down a bit. I have never seen the formula associated with it

**Comment by Starsa** [comment]
**Deep Learning**- I have only read about it. Not as familiar with all the parts

**Comment by CLuiz** [comment]
The accuracy/metrics question is something we will spend time on. It is super important to understand. I always ask at least one question on eval metrics when I interview folks. Same with bias and variance.

I see deep learning as less important at your level, because it is so complex. Lots to remember there. Looking forward to discussing all three.

**Comment by Starsa** [comment]
**Gradient**- 

- cliff
- decent
- clipping
- boosting

I seem to get these all confused.

**Comment by Starsa** [comment]
**Alpha in Ridge Regression**

- Alpha is a hyperparameter
  - used to control the learning process
  - not derived by training
  - select before training or fitting a model
- Ridge Regression is used when multicollinearity occurs (multicollinearity is when there is a strong relationship between independent variables)
  - Used to regularize and reduce standard error to avoid overfitting
- Regression 
  - trains models to predict quantitative (continuous) targets
  -

**Comment by Starsa** [comment]
**Underfitting and Overfitting**

- Underfitting
  - A model is underfit when it fails to capture the pattern in the data
  - It suffers from high bias
- Overfitting
  - Overfitting occurs when a model starts to memorize the aspects of the training data and loses the ability to generalize
  - Fits the noise rather than the data

**Comment by Starsa** [comment]
**Chi- Squared**

- Tests if categorical data shows up at a rate different than random
